posframe.el

```diff
@@ -122,7 +122,7 @@
 
 ;; #+BEGIN_EXAMPLE
 ;; (setq posframe-arghandler #'my-posframe-arghandler)
-;; (defun my-posframe-arghandler (posframe-buffer arg-name value)
+;; (defun my-posframe-arghandler (buf-or-name arg-name value)
```

Comment:
i still like posframe-buffer :-) if you would like to change, i prefer buffer-or-name, for get-buffer use this

Reply:
Yeah, I explained the reasons I don't think "posframe-buffer" is ideal, but "buffer-or-name" is even better, of course. I'll change it, thanks.